web-share/src/svelte/components/Media.svelte

```diff
@@ -0,0 +1,94 @@
+<!--
+Copyright 2019 Google, Inc.
+
+Licensed under the [Apache License, Version 2.0](LICENSE) (the "License");
+you may not use this file except in compliance with the License. You may
+obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<script>
+	import Button from './Button.svelte';
+
+	export let contentType = '';
+	export let src = '';
+
+	const canShare = 'canShare' in navigator;
+
+	function handleView() {
+		window.location.href = src;
```

Comment:
For audio/video files, it downloads file while image shows it inline. Is that intended?

Reply:
No, it wasn't really. I was just looking for a quick way of showing the image in a standalone view, and didn't realize it wouldn't also work for audio and video.

I can add a separate `View.svelte` component that will display any cached media file inline.

Reply:
Yup! Should be able to get to this over the next few days.

Reply:
Added. I might need to tweak the UI a bit, but `/view/...` routes should work now.